_posts/elements/2015-02-04-example-kitchen-sink.md

```diff
@@ -267,71 +296,57 @@ description:  This is a kitchen sink of common HTML elements
                     <option value="2">Option 2</option>
                     <option value="3" disabled>Disabled Option</option>
                 </optgroup>
-            </select>
-        </p>
+            </select></p>
 
         <p><label for="select_element_disabled">Disabled Select Element:</label><br/>
             <select id="select_element_disabled" disabled>
                 <option value="1">Unselectable Option</option>
                 <option value="2">This option should not even be seen</option>
-            </select>
-        </p>
+            </select></p>
 
         <p>Radio Buttons:<br />
             <label><input type="radio" class="radio" name="radio_button" value="radio_1" /> Radio 1</label><br/>
             <label><input type="radio" class="radio" name="radio_button" value="radio_2" /> Radio 2</label><br/>
             <label><input type="radio" class="radio" name="radio_button" value="radio_3" /> Radio 3</label><br/>
-            <label><input type="radio" class="radio" name="radio_button" value="radio_4" disabled /> Radio Disabled</label><br/>
-        </p>
+            <label><input type="radio" class="radio" name="radio_button" value="radio_4" disabled /> Radio Disabled</label><br/></p>
 
         <p>Checkboxes:<br />
             <label><input type="checkbox" class="checkbox" name="checkboxes" value="check_1" /> Checkbox 1</label><br/>
             <label><input type="checkbox" class="checkbox" name="checkboxes" value="check_2" /> Checkbox 2</label><br/>
             <label><input type="checkbox" class="checkbox" name="checkboxes" value="check_3" /> Checkbox 3</label><br/>
-            <label><input type="checkbox" class="checkbox" name="checkboxes" value="check_4" disabled /> Checkbox Disabled</label><br/>
-        </p>
+            <label><input type="checkbox" class="checkbox" name="checkboxes" value="check_4" disabled /> Checkbox Disabled</label><br/></p>
```

Comment:
lol, sorry - I might be having a hard time reading this line/diff of code. Here's what I'd ideally like:

```html
<p>Checkboxes:<br />
    <label>
        <input type="checkbox" class="checkbox" name="checkboxes" value="check_1" />
        Checkbox 1
    </label><br/>
    <label>
        <input type="checkbox" class="checkbox" name="checkboxes" value="check_2" /> 
        Checkbox 2
    </label><br/>
    <label>
        <input type="checkbox" class="checkbox" name="checkboxes" value="check_3" /> 
        heckbox 3
    </label><br/>
    <label>
        <input type="checkbox" class="checkbox" name="checkboxes" value="check_4" disabled /> Checkbox Disabled
    </label><br/>
</p>
```

Again this is just copy/paste code from  a vendor, so ideally we wouldn't have the ``<br />`` elements in anything we've written.